examples/examples/balance_transfer.rs

```diff
@@ -26,23 +25,20 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     tracing_subscriber::fmt::init();
 
     let signer = PairSigner::new(AccountKeyring::Alice.pair());
-
-    let api = ClientBuilder::new()
-        .build()
-        .await?
-        .to_runtime_api::<polkadot::RuntimeApi<DefaultConfig, PolkadotExtrinsicParams<DefaultConfig>>>();
-
-    // Submit the `transfer` extrinsic from Alice's account to Bob's.
     let dest = AccountKeyring::Bob.to_account_id().into();
 
-    // Obtain an extrinsic, calling the "transfer" function in
-    // the "balances" pallet.
-    let extrinsic = api.tx().balances().transfer(dest, 123_456_789_012_345)?;
+    // Create a client to use:
+    let api = OnlineClient::<PolkadotConfig>::new().await?;
```

Comment:
so one doesn't have to call `to_runtime_api()` anymore nice

Reply:
Yup; no more "hard" link between static codegen and API :)